src/sentry/static/sentry/app/components/onboardingWizard/toDoItem.jsx

```diff
@@ -60,76 +63,94 @@ const TodoItem = createReactClass({
     return learnMoreUrl;
   },
 
-  skip: function(task) {
+  recordAnalytics(action) {
+    let org = this.getOrganization();
+    let {task} = this.props;
+    analytics('onboarding.wizard_clicked', {
+      org_id: parseInt(org.id, 10),
+      todo_id: parseInt(task.task, 10),
+      todo_title: task.title,
+      action,
+    });
+  },
+
+  onSkip(task) {
+    this.recordAnalytics('skipped');
     this.props.onSkip(task);
     this.setState({showConfirmation: false});
   },
 
-  render: function() {
+  handleClick(e) {
+    this.recordAnalytics('clickthrough');
+    e.stopPropagation();
+  },
+
+  render() {
+    let {task, className} = this.props;
+    let {showConfirmation} = this.state;
     let learnMoreUrl = this.learnMoreUrlCreator();
     let description;
 
-    switch (this.props.task.status) {
+    switch (task.status) {
       case 'complete':
         description = tct('[user] completed [dateCompleted]', {
-          user: this.props.task.user,
-          dateCompleted: moment(this.props.task.dateCompleted).fromNow(),
+          user: task.user,
+          dateCompleted: moment(task.dateCompleted).fromNow(),
         });
         break;
       case 'pending':
         description = tct('[user] kicked off [dateCompleted]', {
-          user: this.props.task.user,
-          dateCompleted: moment(this.props.task.dateCompleted).fromNow(),
+          user: task.user,
+          dateCompleted: moment(task.dateCompleted).fromNow(),
         });
         break;
       case 'skipped':
         description = tct('[user] skipped [dateCompleted]', {
-          user: this.props.task.user,
-          dateCompleted: moment(this.props.task.dateCompleted).fromNow(),
+          user: task.user,
+          dateCompleted: moment(task.dateCompleted).fromNow(),
         });
         break;
       default:
         description = this.formatDescription();
     }
 
-    let classes = classNames(this.props.className, this.props.task.status, {
-      blur: this.state.showConfirmation,
+    let classes = classNames(className, task.status, {
+      blur: showConfirmation,
     });
 
     let showSkipButton =
-      this.props.task.skippable &&
-      this.props.task.status != 'skipped' &&
-      this.props.task.status != 'complete' &&
-      !this.state.showConfirmation;
+      task.skippable &&
+      task.status != 'skipped' &&
```

Comment:
nit, I know this is existing code, but we should use triple `=` comparisons to be consistent throughout the codebase (I should make this an eslint warning).